### ISSUE TITLE
AssemblyInfoFilename file not found

### ISSUE THREAD

**Issue by mrbiggred** [GitTools/actions#125]
When I try to update the AssemblyInfo.cs files by setting the `updateAssemblyInfo: true` I get the following error:

```
 AssemblyInfoFilename file not found at 
```

Might be related to this [issue](https://github.com/GitTools/GitVersion/issues/2225) on the GitVersion repo.

You can view the error in action at:

https://github.com/saturdaymp/XPlugins.iOS.BEMCheckBox/runs/710150101?check_suite_focus=true

To reproduce:

1) Create a GitHub workflow file with the following:

```yml
name: CI
on: [push]

jobs:
  build:
    name: Build Job
    runs-on: macos-latest
    defaults:
      run:
        working-directory: Source

    steps:
      - name: Checkout
        uses: actions/checkout@v2

      - name: Fetch history for GitVersion
        run: git fetch --prune --unshallow

      - name: Install GitVersion
        uses: gittools/actions/gitversion/setup@v0.9.3
        with:
          versionSpec: '5.3.4'

      - name: Generate Version Info using GitVersion
        id: gitversion
        uses: gittools/actions/gitversion/execute@v0.9.3
        with: 
          updateAssemblyInfo: true
      
      - name: Display GitVersion values
        run: |
          echo "Major: ${{ steps.gitversion.outputs.major }}"
          echo "Minor: ${{ steps.gitversion.outputs.minor }}"
          echo "Patch: ${{ steps.gitversion.outputs.patch }}"
```

2) When run the following error message is displayed:

```
Run gittools/actions/gitversion/execute@v0.9.3
  with:
    updateAssemblyInfo: true
    useConfigFile: false
  env:
    DOTNET_CLI_TELEMETRY_OPTOUT: 1
##[error]Error: AssemblyInfoFilename file not found at 
```

**Comment by BertelBB** [comment]
I had the same issue and I solved it by giving a relative path to the `AssemblyInfo.cs` file, using the `updateAssemblyInfoFilename` variable.

```
...
    - name: Use GitVersion
      id: gitversion
      uses: gittools/actions/gitversion/execute@v0.9.3
      with:
        updateAssemblyInfo: true
        updateAssemblyInfoFilename: ./src/SampleService/Properties/AssemblyInfo.cs
...
```

**Comment by shieldsjared** [comment]
Same error. Dont want to have to specify a path, as I have multiple assemblies in my package that need to be updated/versioned...

**Comment by shieldsjared** [comment]
A temporary workaround for this is to specify `/updateAssemblyInfo` in the **Additional GitVersion arguments** parameter of the Azure Devops Task.  (Huge thanks for this tip on another issue https://github.com/GitTools/GitVersion/issues/2225#issuecomment-624425847)

**Comment by Skoucail** [comment]
Any fix for this issue?
We still have problems in version 5.3.7

**Comment by chris-codeflow** [comment]
The casing of `Filename` is also incorrect and should be changed to `FileName` to be consistent with Microsoft's naming conventions.

**Comment by manicsquirrel** [comment]
> We are still having problems in Azure DevOps (running gitversion 5.7.3).
> 
> @arturcic I believe you did the fix for the gitversion tool itself but forgot to change the Azure DevOps Task.
> Gitversion tool issue: [GitTools/GitVersion#2225](https://github.com/GitTools/GitVersion/issues/2225)
> Bug in Azure DevOps Task:
> 
> https://github.com/GitTools/actions/blob/0333200fa748a48465f646c12992008d01cf777d/src/tools/gitversion/tool.ts#L88-L103

Is there any movement on this? It is still an issue.

**Comment by chris-codeflow** [comment]
I've submitted a PR (see #316 ) to resolve this issue.

**Comment by arturcic** [comment]
Fixed by #316
